run.sh

```diff
@@ -15,17 +15,28 @@ while [ -n "$1" ]; do
   shift
 done
 
-SDC_HOME=${SDC_HOME:-$(mktemp -d)}
-
+if [ -d "$SDC_HOME" ]; then
+  SDC_HOME=${SDC_HOME}
+else
+  SDC_HOME="/tmp/sdc-home"
```

Comment:
note this is not platform independent (mac doesn't use `/tmp` for tempdirs), so I think we want to keep the `mktemp -d` logic